pkgs/top-level/all-packages.nix

```diff
@@ -1945,11 +1945,8 @@ with pkgs;
 
   cool-retro-term = libsForQt5.callPackage ../applications/misc/cool-retro-term { };
 
-  coreutils = callPackage ../tools/misc/coreutils {
-    aclSupport = stdenv.isLinux;
-    attrSupport = stdenv.isLinux;
-  };
-
+  coreutils = callPackage ../tools/misc/coreutils { };
+  coreutils-full = coreutils.override { minimal = false; };
```

Comment:
Would be coreutils-full be part of NixOS?

Reply:
Yes, see the `nixos/*` part of the diff: https://github.com/NixOS/nixpkgs/pull/45720/files#diff-dcba170e079cd72958889058d01a4a02R16 ;-)